### ISSUE TITLE
Add "Save & Launch" button next to "Cancel" & "Save" in the Template Details page

### ISSUE THREAD

**Issue by hiyokotaisa** [ansible/awx#1991]
##### ISSUE TYPE
 - Feature Idea

##### COMPONENT NAME
 - UI

##### SUMMARY

The customer requests to add "Save & Launch" button next to "Cancel" & "Save" in the Template Details page.

It would be extremely helpful and convenient if the "Launch" button is available next to the "Cancel" & "Save" buttons in that same page rather than going through the search process to look for the template again and then launch it from there.

##### EXPECTED RESULTS

A "Save & Launch" button is added right next to the "Cancel" & "Save" buttons in the Template Details page

**Comment by wenottingham** [comment]
This seems like it would be more useful to have the filters persist/change how the list operates.

**Comment by unlikelyzero** [comment]
QE Questions and Concerns:

- Potentially have a permission case to check where you have write but no execute role.
- Edit one and launch another
- WFJT, SJT, JT converted and launched
- Edit but undo and still has the dirty flag
- Save and launch with launch type errors (missing project or org)
- Save and launch with form type errors (duplicate name, missing field, etc)
- How is the user is notified that the job is saved? Refreshed view

**Comment by wenottingham** [comment]
Initial assumptions:

>  Potentially have a permission case to check where you have write but no execute role.

Would be handled by normal can_XYZ permission checking (do not show in this case.)

>  Edit one and launch another

How would you get here?

>    Save and launch with launch type errors (missing project or org)
>    Save and launch with form type errors (duplicate name, missing field, etc)

If save fails, no launch is done.

>  How is the user is notified that the job is saved? Refreshed view

We do have toast notifications, I don't know if from a UX perspective expanding our use of them is  good.
(We could do something like have the launch option *in* the toast notification for a save.)

**Comment by AlanCoding** [comment]
I want to report that the same difficulty of navigation is felt for updating projects. Let me outline the steps to reproduce this scenario specifically:

 - I have 156 projects in my server, for 8 pages in the UI
 - I have ran a job and am on the job details page
   - I just pushed a commit that changes the playbook
   - I need to update the project and relaunch (just once manually, I don't want to mess with update_on_launch)
 - I click on the link to the project, from job details
 - no update button is available

In this case, the project was on page 2. As others noted for JTs, I either have to search for the name of the project, or I have to alphabetically determine what page its on by clicking page-by-page.

A similar "Save & Update" button for projects could resolve the user's difficulty in this specific user story, but any solution that resolves the issue would be great.
